clusterctl/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 The Kubernetes Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: image push dev_image dev_push fix_gcs_permissions
+
+GCR_BUCKET = k8s-cluster-api
+PREFIX = gcr.io/$(GCR_BUCKET)
+DEV_PREFIX ?= gcr.io/$(shell gcloud config get-value project)
+NAME = clusterctl
+TAG = 0.1.0
```

Comment:
I think it would be much better if this tag matched the clusterctl release version. https://github.com/kubernetes-sigs/cluster-api/releases

Reply:
Agreed, I'll change it. Thanks!

I thought 0.1.0 was the [first semver](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase), but that is not strictly true.

Reply:
Was this updated?

Reply:
Yes, sorry for the delay,